format/Flight.proto

```diff
@@ -321,6 +348,13 @@ message FlightEndpoint {
    * represent redundant and/or load balanced services.
    */
   repeated Location location = 2;
+
+  /*
+   * Expiration time of this stream. If present, clients may assume
+   * they can retry DoGet requests. Otherwise, clients should avoid
+   * retrying DoGet requests.
```

Comment:
I wonder if wording it as "it is application-defined whether DoGet requests may be retried" might be better since technically that is how it was before